src/azul/transformer.py

```diff
@@ -176,6 +198,7 @@ def transform(self,
 
         :param uuid: The UUID of the bundle to create documents for
         :param version: The version of the bundle to create documents for
+        :param deleted: Whether or not the bundle being indexed is deleted
```

Comment:
```suggestion
        :param deleted: Whether or not the bundle being indexed has been deleted
```

???

Reply:
changed to "is a deleted bundle" since "has been" isn't as temporally precise.